scripts/clean-dist.mjs

```diff
@@ -0,0 +1,9 @@
+import { rm } from 'fs/promises';
+import { dirname, resolve } from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const distDir = resolve(__dirname, '..', 'dist');
+
+await rm(distDir, { recursive: true, force: true });
```

Comment:
Literally why... Just use rimraf?

Reply:
Because that would be an extra dev dependency

Reply:
Yes, an entire dependency for literally 3 lines of code, lol.

Reply:
3 lines of code. That's cute. Try 1 line of code lul.